app/components/card-hazard.tsx

```diff
@@ -66,61 +50,65 @@ const CardHazard: React.FC<CardHazardProps> = ({
   };
 
   return (
-    <Card flex={1} maxW={400} p={{ base: "16px", md: "20px" }}>
-      <Popover
-        placement="bottom"
-        returnFocusOnClose={false}
-        closeOnBlur={true}
+    <Card.Root flex={1} maxW={400} p={{ base: "16px", md: "20px" }}>
```

Comment:
many Chakra components are now referred to with ".Root" and their children by e.g. ".Item"